packages/ra-ui-materialui/src/input/DateInput.stories.tsx

```diff
@@ -45,10 +68,16 @@ export const Validate = () => (
 
 const i18nProvider = polyglotI18nProvider(() => englishMessages);
 
-const Wrapper = ({ children }) => (
+const Wrapper = ({
+    children,
+    simpleFormProps,
+}: {
+    children: React.ReactNode;
+    simpleFormProps?: Partial<SimpleFormProps>;
+}) => (
     <AdminContext i18nProvider={i18nProvider} defaultTheme="light">
         <Create resource="posts">
-            <SimpleForm>
+            <SimpleForm {...simpleFormProps}>
                 {children}
                 <FormInspector name="published" />
```

Comment:
issue: some stories are using `published`, while others are using `publishedAt`, so the `FormInspector` doesn't always work. Can you fix it?